lambdas/process-cri-callback/src/main/java/uk/gov/di/ipv/core/processcricallback/service/CriCheckingService.java

```diff
@@ -184,7 +184,8 @@ public void validateCallbackRequest(
                 || !state.equals(criOAuthSessionItem.getCriOAuthSessionId())) {
             throw new InvalidCriCallbackRequestException(ErrorResponse.INVALID_OAUTH_STATE);
         }
-        if (configService.getOauthCriActiveConnectionConfig(criId) == null) {
+        if (configService.getOauthCriActiveConnectionConfig(callbackRequest.getCredentialIssuer())
+                == null) {
             throw new InvalidCriCallbackRequestException(
                     ErrorResponse.INVALID_CREDENTIAL_ISSUER_ID);
         }
```

Comment:
Similarly - is this check still valid?

Reply:
I'm not sure if that would ever return null tbh - wouldn't we get a `NoConfigForConnectionException` if the config wasn't found for the CRI?

Reply:
probably! - even more reason to use the CRI enum as the validation mechanism :)